doc/api/crypto.md

```diff
@@ -1378,6 +1380,8 @@ For private keys, the following encoding options can be used:
 When PEM encoding was selected, the result will be a string, otherwise it will
 be a buffer containing the data encoded as DER.
 
+When JWK encoding was selected, all other encoding options are ignored.
```

Comment:
It might be a good idea to at least make 'JWK' here a link here to a specification or some other useful site for those who don't know what it is or even what it stands for.

Reply:
✅